src/cascadia/TerminalControl/TermControl.cpp

```diff
@@ -499,9 +501,9 @@ namespace winrt::Microsoft::Terminal::TerminalControl::implementation
         }
 
         _root.PointerWheelChanged({ this, &TermControl::_MouseWheelHandler });
-        _root.PointerPressed({ this, &TermControl::_MouseClickHandler });
-        _root.PointerMoved({ this, &TermControl::_MouseMovedHandler });
-        _root.PointerReleased({ this, &TermControl::_PointerReleasedHandler });
+        _swapChainPanel.PointerPressed({ this, &TermControl::_PointerPressedHandler });

```

Comment:
I'd maybe add a comment referring to this issue that calls out why these are on `_swapChainPanel` instead of `_root` like (most) everything else